src/test/java/com/exasol/adapter/dialects/elasticsearch/ElasticSearchSqlDialectIT.java

```diff
@@ -67,10 +71,11 @@ private static ElasticsearchContainer createElasticsearchContainer() {
         final ElasticsearchContainer container = new ElasticsearchContainer(ELASTICSEARCH_DOCKER_IMAGE_REFERENCE);
         // Disable TLS for Elasticsearch server as the self-signed certificate uses the wrong hostname and we can't
         // configure a HostnameVerifier for the JDBC driver.
-        container.withEnv("xpack.security.enabled", "false");
-        container.withEnv("discovery.type", "single-node");
+        container.withEnv("ES_SETTING_XPACK_SECURITY_ENABLED", "false");
+        container.withEnv("ES_SETTING_DISCOVERY_TYPE", "single-node");
```

Comment:
See https://www.elastic.co/guide/en/elasticsearch/reference/7.17/docker.html#docker-configuration-methods how to configure ES in Docker via env variables